minutes/minutes-2015-02-19.md

```diff
@@ -0,0 +1,147 @@
+## Minutes
+
+### Attending
+
+*   Aleksandra Pawlik
+*   Greg Wilson
+*   Jason Williams
+*   Katy Huff
+*   Matt Davis
+*   Raniere Silva
+
+### Apologies
+
+*   Adina Howe
+*   Karin Lagesen
+
+### Actions/Resolutions
+
+*   GW: blog post about committees which need more members:
+    *   Finance does not need to be included. it is full.
+    *   Assessment is somewhat full, but could be included in the blog post
+    *   Fundraising
+    *   Mentorship
+    *   Lessons probably not needed in blog post.
+    *   Communications (contact Adina)
+*   GW: Contact Camille Avestruz to check back in about her diversity efforts
+    and see what the SC can do to help out!
+*   JW/KH: get started with Lucid Meetings.
+
+### Old business:
+
+*   Approval of minutes
+    *   https://github.com/swcarpentry/board_pvt/blob/master/minutes/minutes-2015-02-12.md 
```

Comment:
On Mon, Mar 30, 2015 at 11:12:55AM -0700, Katy Huff wrote:
> Thank you @wking .

No problem.  Sorry I missed the merge window ;).